index.rst

```diff
@@ -42,6 +42,8 @@ The requirement specifies that the rms repeatability of the mirror positioning i
 .. image:: images/797_histogram_position_rotation_xyz.png
   :width: 700px
 
+The requirement specifies that the rms repeatability of the mirror positioning is what we need. Plot the rms for each of the distributions shown in the violin plot
```

Comment:
Define "rms" and clarify what you mean with "rms repeatability." Does this mean it has to be the same value every time? What is the tolerance?